data/governments.txt

```diff
@@ -1439,6 +1541,22 @@ government "Sheragi"
 	"crew attack" 0
 	"crew defense" 0
 
+government "Silent Ones"
+	"display name" "???"
+	swizzle 0
+	"player reputation" 1
```

Comment:
This could be 0, correct, because you can’t lose reputation with them?
```suggestion
	"player reputation" 0
```

Reply:
no it is 1 because they like you unless you attack them

Reply:
But you have custom penalties so they will still like you even if you attack them, after a day passes.

Reply:
yes, spacelife will not remember you